packages/nx-sonarqube/src/executors/scan/schema.json

```diff
@@ -45,6 +45,24 @@
       "type": "string",
       "default": "300"
     },
+    "skipDependencyTypes": {
```

Comment:
Please update README.md in the root of the project with these properties and remove `skipImplicitDeps` property from the markdown as well